examples/code/home/configuration/engine/full_config.1.0.yaml

```diff
@@ -0,0 +1,125 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/ArcadeAI/schemas/main/engine/config/1.0/schema.json
```

Comment:
This file was renamed to add the version `1.0` to the filename. (so that later we can have multiple versions/examples coexisting)